src/comps/temp-withdraw.tsx

```diff
@@ -0,0 +1,244 @@
+"use client";
+
+import { useState } from "react";
+import { FlowContainer } from "./core/FlowContainer";
+import { FlowFormDynamic, NameKeysInfo } from "./core/Form";
+import { Heading, SubText } from "./core/Heading";
+import LandingAnimation from "./core/LandingAnimation";
+import { DefaultNetworkConfigurations } from "@leather.io/models";
+import { useRouter } from "next/navigation";
+
+import { bytesToHex, hexToBytes } from "@stacks/common";
+
+import * as bitcoin from "bitcoinjs-lib";
+import { useAtomValue } from "jotai";
+import { bridgeConfigAtom, walletInfoAtom, WalletProvider } from "@/util/atoms";
+import {
+  bufferCV,
+  makeUnsignedContractCall,
+  PostConditionMode,
+  serializeTransaction,
+  tupleCV,
+  uintCV,
+} from "@stacks/transactions";
+import { serverBroadcastTx } from "@/actions/server-broadcast-tx";
+import {
+  callContractAsigna,
+  callContractFordefi,
+  callContractLeather,
+  callContractXverse,
+} from "@/util/wallet-utils";
+import { getStacksNetwork } from "@/util/get-stacks-network";
+
+const decodeBitcoinAddressToClarityRecipient = (
+  address: string,
+  network: DefaultNetworkConfigurations,
+) => {
+  const lower = address.toLowerCase();
+  const isMainnet = network === "mainnet";
+
+  // check if passed in address is bech32
+  let decodedBech32;
+  let isBech32 = false;
+  if (
+    lower.startsWith("bc1") ||
+    lower.startsWith("bcrt1") ||
+    lower.startsWith("tb1")
```

Comment:
I think we don't want testnet addresses (`tb1`)

Reply:
done